EventFilter/CSCRawToDigi/src/CSCDDUEventData.cc

```diff
@@ -277,7 +271,6 @@ void CSCDDUEventData::unpack_data(const uint16_t *buf, CSCDCCExaminer* examiner)
 
     }
     theSizeInWords = dduBufSize+12;
-    buf=inputBuf+dduBufSize;
```

Comment:
If the intention was to modify the pointer passed to this function, then it should have been passed by reference. 
Otherwise, I agree with the static analyzer that this is a dead assignment as such, and it can be removed